### ISSUE TITLE
Proposition: store blocks in less files

### ISSUE THREAD

**Issue by ejpcmac** [input-output-hk/cardano-sl#2224]
As far as I know, blocks are currently stored in individual files. There are two main drawbacks doing this:

* it is really slow to backup,
* it adds a huge overhead, because most file systems store a file in an integer number of sectors.

An idea would be to store the blocks in bigger files with an index, like Bitcoin Core does. It would avoid the overhead and speed up backups. It would create these files incrementally in order to avoid old files to change and facilitate backups.

**Comment by gromakovsky** [comment]
Yes, there is a proposal to do it: https://github.com/input-output-hk/cardano-sl/pull/2169

**Comment by ejpcmac** [comment]
Ah, great. I’d checked for any related issue, but not in the proposals.
